backend/x86_binary_emitter.ml

```diff
@@ -663,6 +663,12 @@ let emit_andpd b dst src =
       emit_mod_rm_reg b 0 [ 0x0f; 0x54 ] rm (rd_of_regf reg)
   | _ -> assert false
 
+let emit_andps b dst src =
```

Comment:
The only differences between the existing
instructions and the new ones are the
prefixes; should we factor out the common
parts?

Reply:
Factored